pandas/core/groupby/generic.py

```diff
@@ -646,6 +647,21 @@ def nunique(self, dropna: bool = True) -> Series | DataFrame:
         a    2
         b    1
         dtype: int64
+
+        For Resampler:
+
+        >>> ser = pd.Series([1, 2, 3, 3], index=pd.DatetimeIndex(
+        ...                 ['2023-01-01', '2023-01-15', '2023-02-01', '2023-02-15']))
+        >>> ser
+        2023-01-01    1
+        2023-01-15    2
+        2023-02-01    3
+        2023-02-15    3
+        dtype: int64
+        >>> ser.resample('M').nunique()
```

Comment:
the examples look good!

any chance we could make an example which uses 'D' here instead please, so as to not add even more files to update in #52064?

Reply:
Changed all to `MS` to avoid too many changes, as per our conversation.